account_journal_report/wizard/wizard_print_journal_ledger_view.xml

```diff
@@ -0,0 +1,64 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<odoo>
+    <record id="account_journal_ledger_report_view" model="ir.ui.view">
+        <field name="name">Print journal ledger</field>
+        <field name="model">account.journal.entries.report</field>
+        <field name="arch" type="xml">
+            <form string="Print journal ledger">
+                <group>
+                    <group>
+                        <field name="sort_selection"/>
+                    </group>
+                    <group>
+                        <field name="landscape"/>
+                    </group>
+                </group>
+                <group name="dates">
+                    <field name="date_start"/>
+                    <field name="date_end"/>
+                </group>
+                <group/>
+                <label for="journal_ids"/>
+                <field name="journal_ids" nolabel="1" options="{'no_create': True}"/>
+                <group/>
+                <footer>
+                    <button name="print_report" string="Print" type="object"
+                            default_focus="1" class="oe_highlight"/>
+                    or
+                    <button special="cancel" string="Cancel"
+                            class="oe_link"/>
+                </footer>
+            </form>
+        </field>
+    </record>
+
+    <record id="wizard_account_journal_ledger_report_new"
+            model="ir.actions.act_window">
+        <field name="name">Print Journal Ledger</field>
+        <field name="type">ir.actions.act_window</field>
+        <field name="res_model">account.journal.entries.report</field>
+        <field name="view_type">form</field>
+        <field name="view_mode">form</field>
+        <field name="view_id" ref="account_journal_ledger_report_view"/>
+        <field name="target">new</field>
+    </record>
+
+    <!-- Add XLSX export button -->
+    <record id="account_journal_ledger_xlsx_view" model="ir.ui.view">
```

Comment:
The export button should be added in the main view, there is no need to creat an extension view for it.